pyperformance/data-files/benchmarks/bm_alloc/run_benchmark.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
+"""
+Allocate blocks of memory with bytearray
+
+This benchmark stresses memory allocator
+"""
+
+import pyperf
+
+
+ONE_MB = 1024 * 1024
+
+DEFAULT_SIZES = [32, 128, 512, 8192, ONE_MB, 8 * ONE_MB]
```

Comment:
This looks good to me, but take into account that `obmalloc` will trigger only for blocks less than 512:

https://github.com/python/cpython/blob/b5527688aae11d0b5af58176267a9943576e71e5/Objects/obmalloc.c#L889

We could go as it is and test globally all allocators (but we would be benchmarking `malloc`, basically) or restrict this test to small allocations.

Reply:
I have updated the test to use smaller sizes in order to stress obmalloc. We can add another benchmark later to test libc malloc vs. mimalloc.